src/utah-design-system/RadioGroup.jsx

```diff
@@ -0,0 +1,78 @@
+import * as RadixRadioGroup from '@radix-ui/react-radio-group';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+export default function RadioGroup({
+  ariaLabel,
+  items,
+  defaultValue,
+  value,
+  onValueChange,
+}) {
+  return (
+    <RadixRadioGroup.Root
+      aria-label={ariaLabel}
+      className="flex flex-col items-start"
+      defaultValue={defaultValue}
+      value={value}
+      onValueChange={onValueChange}
+    >
+      {items.map((item) => {
+        const id = `${ariaLabel}-${item.value}`;
+
+        return (
+          <div className="group flex items-center" key={id}>
+            <div
+              className={clsx(
+                'rounded-full p-1',
+                !item.disabled &&
+                  'group-hover:bg-slate-200 group-focus:bg-slate-200 group-active:bg-slate-200'
+              )}
+            >
+              <RadixRadioGroup.Item
+                className={clsx(
+                  'flex h-4 w-4 items-center justify-center rounded-full border border-slate-500 bg-white',
+                  item.disabled &&
+                    '!border-slate-300 !bg-slate-50 data-[state=checked]:!bg-slate-300 data-[state=checked]:!text-white'
+                )}
+                value={item.value}
+                id={id}
+                disabled={item.disabled}
+              >
+                <RadixRadioGroup.Indicator
+                  className={clsx(
+                    'after:block after:h-4 after:w-4',
+                    'after:rounded-full after:content-[""]',
+                    'after:border-4 after:border-slate-500'
+                  )}
+                />
+              </RadixRadioGroup.Item>
+            </div>
+            <label
+              htmlFor={id}
+              className={clsx(
+                'ml-1 cursor-pointer leading-5',
+                item.disabled && '!text-slate-300'
```

Comment:
**suggestion**: refactor to remove !

Reply:
Resolved in dcdb5f79f0d04ca9f35c62e740fb7e4b3ebd5272.